src/elements/transformer/DefaultTransform.ts

```diff
@@ -0,0 +1,47 @@
+import type { FieldNameToValueMap } from "../../plugin/fieldViews/helpers";
```

Comment:
Files that aren't JSX components shouldn't need capital letters ... although we might be horribly inconsistent elsewhere 😬

Reply:
Yep good spot, I'll do make that change